sys/include/net/ipv6/addr.h

```diff
@@ -759,14 +771,16 @@ static inline int ipv6_addr_split_prefix(char *addr_str)
 /**
  * @brief split IPv6 address + interface specifier
  *
- * E.g., "fe80::1%5" returns "5", changes @p addr_str to "fe80::1"
+ * E.g., "fe80::1%if5" returns a pointer to "if5", changes @p addr_str to
+ * "fe80::1"
```

Comment:
The original documentation was also reflecting the new behavior (not the old behavior though ;-)). Do we really need to change it?

Reply:
I just wanted to make it more obvious, but we can keep it.

Reply:
I'm not fighting against the change. Just pointed out that the original documentation was already correct. But maybe you are right. The new one makes it more clear.